cookbooks/aws-parallelcluster-environment/attributes/environment.rb

```diff
@@ -37,6 +37,7 @@
 default['cluster']['efs_fs_ids'] = ''
 default['cluster']['efs_encryption_in_transits'] = ''
 default['cluster']['efs_iam_authorizations'] = ''
+default['cluster']['efs_accesspoint_ids'] = ''
```

Comment:
This is a customer facing feature. May you please add an entry in the CHANGELOG?